cocos/ui/UIEditBox/UIEditBoxImpl-win32.cpp

```diff
@@ -545,12 +529,12 @@ bool EditBoxImplWin::initWithSize(const Size& size)
 
 void EditBoxImplWin::setFont(const char* pFontName, int fontSize)
 {
-	if(_label != nullptr) {
+	if (_label != nullptr) {
```

Comment:
Could you put the left curl bracket in a separate line?  It's better to keep all the style the same with cocos2d-x code base.  And it's applied to all of your changes. Thanks.